src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.exception.DukeException;
+
+import duke.taskTypes.Task;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Command that contains details when deleting a task
+ */
+public class DeleteCommand extends Command {
+    private final String chosenList;
+
+    /**
+     * Basic Constructor
+     *
+     * @param storage Storage object to save
+     * @param taskList Tasklist to add task to
+     * @param ui Ui to display msg
+     * @param chosenList contains the string that describes which task to be deleted
+     */
+    public DeleteCommand(Storage storage, TaskList taskList, Ui ui, String chosenList) {
+        super(storage, taskList, ui);
+        this.chosenList = chosenList;
+    }
+
+    /**
+     * Executes a set of instructions
```

Comment:
Maybe you can say what exactly the instructions are
Same goes for the other commands!